src/StreamJsonRpc/JsonMessageFormatter.cs

```diff
@@ -135,11 +165,19 @@ public Version ProtocolVersion
         /// <summary>
         /// Gets the <see cref="Newtonsoft.Json.JsonSerializer"/> used when serializing and deserializing method arguments and return values.
         /// </summary>
-        public JsonSerializer JsonSerializer { get; } = new JsonSerializer()
+        public JsonSerializer JsonSerializer { get; }
+
+        /// <inheritdoc/>
+        public JsonRpc Rpc
         {
```

Comment:
Make this an explicit interface implementation so that folks are much less likely to try to set this property. You can do this by changing this line to: `JsonRpc IJsonRpcInstanceContainer.Rpc`